Makefile

```diff
@@ -70,9 +70,10 @@ test-skopeo:
 	@export GOPATH=$$(mktemp -d) && \
 		skopeo_path=$${GOPATH}/src/github.com/containers/skopeo && \
 		vendor_path=$${skopeo_path}/vendor/github.com/containers/image && \
-		git clone -b $(SKOPEO_BRANCH) https://github.com/$(SKOPEO_REPO) $${skopeo_path} && \
-		rm -rf $${vendor_path} && cp -r . $${vendor_path} && rm -rf $${vendor_path}/vendor && \
-		cd $${skopeo_path} && \
+		git clone https://github.com/$(SKOPEO_REPO) $${skopeo_path} && \
+		cd $${skopeo_path} && git checkout $(SKOPEO_BRANCH) && \
+		sed -i 's/FROM fedora/FROM fedora:30/g' Dockerfile && \
+		rm -rf $${vendor_path} && cp -r /gopath/src/github.com/containers/image $${vendor_path} && rm -rf $${vendor_path}/vendor && \
```

Comment:
Does anything change about what this line does? If not, wrapping the `git checkout` in a subshell `(cd … && git checkout ) && ` should allow not changing this line, to make it clearer that nothing is going on, and avoid hard-coding the container layout in this `Makefile`.